src/score_hv/hv_registry.py

```diff
@@ -8,32 +8,32 @@
 from collections import namedtuple
 from score_hv.harvesters.innov_netcdf import InnovStatsCfg, InnovStatsHv
 from score_hv.harvesters.obs_log import ObsInfoCfg, ObsInfoHv
+from score_hv.harvesters.inc_logs import LogIncCfg, LogIncHv
 
 NAMED_TUPLES_LIST = 'tuples_list'
 PANDAS_DATAFRAME = 'pandas_dataframe'
 
 INNOV_NETCDF = 'innov_stats_netcdf'
 OBS_INFO_LOG = 'obs_info_log'
 
```

Comment:
completely nitpick but this blank line should be removed and all the harvester name variables together

Reply:
corrected as suggested